src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -335,4 +341,23 @@ private function configureStorage(array $config, ContainerBuilder $container, Lo
 
         $container->setAlias('sulu_media.storage', 'sulu_media.storage.' . $storage)->setPublic(true);
     }
+
+    private function getSupportedMimeTypes($ghostScriptPath, $ffmpegBinary, $ffprobeBinary)
+    {
+        $mimeTypes = ['image/*'];
+
+        if ($ffmpegBinary
+            && $ffprobeBinary
+            && shell_exec('which ' . $ffmpegBinary . ' > /dev/null')
```

Comment:
For windows the commands to check if ghostscript or ffmpeg is available are the following:

```cmd
where.exe ffmpeg.exe /Q
where ffmpeg.exe /Q
where.exe "C:\ffmpeg\bin:ffmpeg.exe"
```

```cmd
where.exe gswin64.exe /Q
where.exe "C:\Program Files\gs\gs9.50\bin:gswin64.exe" /Q
```

Attention if search in a specific path the `path` is splitted with `:` not with `\`.

`\Q` means no output and we need the exit code if it is found > /dev/null is not supported by windows and will output the warning shown in the CI currently.

So the correct way on windows could be:

```php
exec('where.exe "C:\Program Files\gs\gs9.50\bin:gswina64.exe"', $output, $exitCode);
$ghostScriptAvailable = (0 === $exitCode);
```

Reply:
The `-q` option shows an error in linux... But at least I know now where the error origins from... I think there is no other way than building an `if` to be able to build this in two different ways...